src/vs/workbench/browser/parts/panel/panelPart.ts

```diff
@@ -798,7 +798,6 @@ export abstract class BasePanelPart extends CompositePart<PaneComposite> impleme
 					name: cachedPanel.name,
 					order: cachedPanel.order,
 					pinned: cachedPanel.pinned,
-					badgeEnabled: cachedPanel.badgeEnabled,
 					visible: !!compositeItems.find(({ id }) => id === cachedPanel.id)
```

Comment:
I'm surprised this goes away?

Reply:
It seems like the information is stored below in the cached panel so why don't we have it here?